### PR TITLE
add k8s.io/examples=master repo and enable [Feature:Example] test

### PR DESCRIPTION
**Pull request by cadmuxe**
this is for https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/example_cluster_dns.go

### CODE REVIEW

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml [hunk 1]
```diff
@@ -599,14 +599,15 @@ periodics:
     - args:
       - "--timeout=340"
       - "--repo=k8s.io/kubernetes=release-1.11"
+      - "--repo=k8s.io/examples=master"
```
Comment (cadmuxe) [inline config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: @krzyzacy can we checkout multiple repo?

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml [hunk 2]
```diff
@@ -648,6 +649,7 @@ periodics:
     - args:
       - "--timeout=340"
       - "--repo=k8s.io/kubernetes=release-1.11"
```
Comment (krzyzacy) [inline config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: I think you don't really need the kubernetes repo for this job

File: config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml (hunk 1 above)
Comment (krzyzacy) [reply config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml]: yes you can

**Comment by krzyzacy** [comment]
the dir structure will be resolved to
```
/workspace
  /k8s.io
    /examples
      /kubernetes
        /(kubernetes binaries)
```

will this work with your code?

**Comment by jingax10** [comment]
@krzyzacy The additional repo intends to address "error: the path "/workspace/k8s.io/kubernetes/kubernetes/examples/cluster-dns/dns-backend-rc.yaml" does not exist". However, the desired path should be /workspace/k8s.io/kubernetes/examples/staging/cluster-dns/dns-backend-rc.yaml it seems different from what you expect. @cadmuxe could you please confirm it?

**Comment by krzyzacy** [comment]
I mean k8s.io should resolve to kubernetes/kubernetes, it won't be k8s.io/kubernetes/kubernetes

**Pull request merged**